### ISSUE TITLE
Use of "Release-As:" in commit 

### ISSUE THREAD

**Issue by MxNxPx** [google-github-actions/release-please-action#952]
### TL;DR

Per [the docs](https://github.com/googleapis/release-please?tab=readme-ov-file#how-do-i-change-the-version-number), previously I could force a version by adding `Release-As: 0.2.0` somewhere in the body of the commit message used in the squash-merge to main regardless of other content in the message.  Now it seems to only work if `Release-As: 0.2.0` is not accompanied by other Github things like co-authors or perhaps just any presence of `---` in the commit message.

### Expected behavior

The GHA workflow run would create/update the release-please PR with the version provided in the `Release-As:` line.

### Observed behavior

It does not set the provided version.

### Action YAML

```yaml
on:
  push:
    branches:
      - main

name: release-please-workflow

jobs:
  tag-new-version:
    permissions: write-all
    runs-on: ubuntu-latest
    outputs:
      release_created: ${{ steps.release-flag.outputs.release_created }}
      release_tag: ${{ steps.release-tag.outputs.release_tag }}
    steps:
      - name: Create Release Tag
        id: tag
        uses: google-github-actions/release-please-action@v4.0.2
      - id: release-flag
        run: echo "release_created=${{ steps.tag.outputs.release_created || false }}" >> $GITHUB_OUTPUT 
      - id: release-tag
        run: echo "release_tag=${{ steps.tag.outputs.version || false }}" >> $GITHUB_OUTPUT 
      - id: release-debug
        run: echo "$GITHUB_CONTEXT"

  publish-release:
    needs: tag-new-version
    if: ${{ needs.tag-new-version.outputs.release_created == 'true'}}
    runs-on: ubuntu-latest
    steps:
      - id: publish-debug
        run: echo "$GITHUB_CONTEXT"
      - name: Publish Release Artifact
        id: echo-release
        run: echo "RELEASE ARTIFACT WILL BE CREATED AT THIS POINT"
```

### Log output

```text
no useful log info
```

### Additional information

workflow:  https://github.com/MxNxPx/quick-test/blob/main/.github/workflows/release-workflow.yaml

commits to main:  https://github.com/MxNxPx/quick-test/commits/main/

**Comment by coderkakarrot** [comment]
I am also facing the same issue.

**Comment by coaxial** [comment]
I can't even get it to release with this commit (verbatim):

```
$ git show 55ea888
commit 55ea888ad48c7bd674c1bbe9673390c939863bb3 (origin/master, origin/HEAD, master)
Author: coaxial <redacted@domain.tld>
Date:   Fri Mar 28 16:12:13 2025 +0100

    chore: release 0.11.0

    Release-As: 0.11.0
```

Did you manage to get this forced-release commit system to work after all?

**Comment by Dannymx** [comment]
Trying to force a version with a commit and doesn't work at:
```
git commit --allow-empty -m "chore: release 1.2.0" -m "Release-As: 1.2.0"
```
